manticore/platforms/linux.py

```diff
@@ -263,12 +264,21 @@ class Linux(Platform):
     This class emulates the most common Linux system calls
     '''
 
-    def __init__(self, program, argv=None, envp=None):
-        '''
+    def __init__(self, program, argv=None, envp=None, **options):
+        r'''
         Builds a Linux OS platform
         :param string program: The path to ELF binary
         :param list argv: The argv array; not including binary.
         :param list envp: The ENV variables.
+        :param \**options: See below
+
+        :Keyword Arguments:
+         * *stack_top* (``int``) --
+           Set the top of the stack to the specified value
+         * *interpreter_base* (``int``) --
+           Set the loading address of the interpreter (if not specified by ELF)
+         * *stack_offset* (``int``) --
```

Comment:
stack_offset... If both versions of the program/emulation use getenv/getauxv the trace will diverge. Consider making the stacks match instead of making a gap in one? The gap is also achievable via a fake envp.